R/convergence.R

```diff
@@ -322,41 +321,6 @@ ess_mean.rvar <- function(x, ...) {
   summarise_rvar_by_element_with_chains(x, ess_mean, ...)
 }
 
-#' Effective sample size for the standard deviation
```

Comment:
You seem to have removed the `ess_sd` function. Was that on purpose? We cannot just remove previously exported functions even if we don't use them anymore in the mcse computation. 

I suggest we add them back and only change the `mcse_sd` function.

Reply:
I put it back with the formula used in mcse_sd

Reply:
Thank you!